lib/charms/observability_libs/v1/kubernetes_service_patch.py

```diff
@@ -59,7 +59,7 @@ def __init__(self, *args):
 
 ```python
 # ...
-from charms.observability_libs.v0.kubernetes_service_patch import KubernetesServicePatch
+from charms.observability_libs.v1.kubernetes_service_patch import KubernetesServicePatch
```

Comment:
I suppose the question is still "is this enough of an API change to really need v1"? It seems like it's still the same for consumers and is just a bugfix

Reply:
It's already v1. This is just a doc fix.